src/main/java/org/stuartgunter/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -463,4 +494,15 @@ public Cluster build(MetricRegistry metrics, HealthCheckRegistry healthChecks) {
 
         return cluster;
     }
+
+    private void validateDnsContactPointsType() {
+        if (DNS.equals(contactPointsType) && contactPoints.length > 1) {
+            throw new ValidationException("Cannot specify more than one contact point when contactPointsType is DNS.");
+        }
+    }
+
+    public enum ContactPointsType {
+        IP,
```

Comment:
I can't remember if the PR was changed since this comment, but I don't think it has. Please could you confirm?

If you're happy, I can make the change as per the previous comments and send them to you for review? What do you think?